Firestore/Swift/Source/Result/DocumentReference+Result.swift

```diff
@@ -0,0 +1,247 @@
+/*
+ * Copyright 2019 Google
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import FirebaseFirestore
+
+#if swift(>=5.0)
+
+  extension DocumentReference {
+    /// Writes the document referred to by this `DocumentReference` with the specified data.
```

Comment:
These docs differ slightly from the [public docs](https://firebase.google.com/docs/reference/swift/firebasefirestore/api/reference/Classes/DocumentReference#setdata_:completion:) in some minor ways:

  * "with the specified data" in this comment
  * `merge` has a different description "Indicates if provided data has to be merged into any existing document"
  * `mergeFields` differs (notably missing "Fields can contain dots to reference nested fields within the document")

Please make these consistent with the other methods; ideally using exactly the same language. Note that using the Swift forms of the type names is appropriate--we've got a bug filed somewhere to change the docs generated from Objective-C sources to this form too.

If you've copied from some other location, please let me know where so that we can make them all consistent.

Reply:
- I added "...with the specified data" to make it consistent with the encodable method where is written "...encoding an instance of encodable".
- Fixed second point.
- Strange but the string wasn't missing
 Waiting reply for the first point to proceed changing it.